tools/cpplint.py

```diff
@@ -527,6 +527,9 @@
 
 _NULL_TOKEN_PATTERN = re.compile(r'\bNULL\b')
 
+_RIGHT_LEANING_POINTER_PATTERN = re.compile(r'[^=|(sizeof)(,\s);&?:]\s'
+                                            r'\*[a-zA-z_]+')
```

Comment:
Correct me if I'm wrong but it looks like this matches `x *p` but not `s *p`, `i *p`, or even `xx *p`.  It looks like it would also be fooled by code like below because it matches `}\n*p`.
```c
if (/*...*/) {
  // ...
}
*p = 42;
```